region/maridia/inner-yellow/Kassiuz Room.json

```diff
@@ -169,11 +169,27 @@
               "canSpringBallJumpMidAir",
               "canUseFrozenEnemies"
             ]}
-          ]},
-          "canTrickySpringBallJump"
+          ]}
         ]}
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Springball",
+      "requires": [
+        "canTrickySpringBallJump",
+        "canCrouchJump",
```

Comment:
The crouch isnt necessary

Reply:
Without the crouch its pretty much a max height jump.  Since there are options for figuring out how to gain extra height, perhaps we can leave it out as a requirement.